dist/0.2.1/mavo.es5.js

```diff
@@ -6204,9 +6204,15 @@ Mavo.Locale.register("en", {
         "month": /^[Y\d]{4}-[M\d]{2}$/i,
         "time": /^[H\d]{2}:[M\d]{2}/i,
         "datetime-local": /^[Y\d]{4}-[M\d]{2}-[D\d]{2} [H\d]{2}:[Mi\d]{2}/i,
-        "date": /^[Y\d]{4}-[M\d]{2}-[D\d]{2}$/i
+        "date": /^[Y\d]{4}-[M\d]{2}-[D\d]{2}$/i,
+        "duration": /^P/,
+        "duration-normal": /(day[s]?|hour[s]?|minute[s]?|second[s]?)$/,
+        "duration-abbreviation": /(d|h|m|s)$/,
       },
       defaultFormats: {
+        "duration": name => "[getduration(".concat(name,"\"officialDurationString\")]"),
+        "duration-abbreviation": name => "[getduration(".concat(name,"\"abbreviatedDurationString\")]"),
+        "duration-normal": name => "[getduration(".concat(name,"\"normalDurationString\")]"),
```

Comment:
We never touch the files in version folders, only the ones directly in `dist`, which are built automatically by gulp.